### ISSUE TITLE
Windows 10 breaks with Etherminer 0.11.0 and 0.12.0 ver

### ISSUE THREAD

**Issue by Andriuzz66** [ethereum-mining/ethminer#409]
Hi to all

Have issue and cant resolve it. Already mining for some time but still getting the same issue. Starting Etherminer it mines for a while and then breaks (miner it self). Also 3-4 time a day when mining Windows 10 stops working for no reason and have to restart mining rig. Have 12pcs. Gtx GeForce 1070 8 Gb, i7 intel, 32Gb RAM, 120 Gb SSD.

Question. Do I have only this problem or is it common? maybe someone has solutions or suggestions.

Appreciate.

**Comment by paucoma** [comment]
Hi, I have read in some issues about the Nvidia cards jumping from State P2 to P0 which causes a jump in memory clock of +200MHz and appears to cause stability issues, depending on the memory "quality". I don't recall which issue number this is under but possible solutions to this would be to force to operate on state P0 and reduce memory overclocking by 200 MHz (since state P0 by default boosts the memory CLK 200MHz)
I have one GTX 1070 and 2 1060s from Asus at the moment on windows 10, i7 7700, 16GB ,and it has been running smoothly for a week. With the GPU tweak app, I have the memory running at 7998 MHz with a memory boost of 8408 and the GPUs around 1450Mhz with a power target of 70% on the 1070 and 60% on the 1060s.. 
What are your GPU settings?

**Comment by Andriuzz66** [comment]
Hi
thx for comment. I havent changed any GPU setting at all. Overclocking is not working with more than 8 cards. I have tries Afterburner it just  not starting. so I havent made any changes. but it breaks very ofte. If I missunderstood Your question refrase it?

BR

**Comment by paucoma** [comment]
Aha, interesting. 
My question was regarding overclocking.. so you have answered that already. I plan on having more than 8 cards in the future so I will see what happens then.. 
Which motherboard do you have? 
Perhaps you could try to limit the pci-e communication speed from gen 3 to gen 2 in the bios. Afaik it should not influence mining speed. 
Another thing to check would be your power connections. Are you using single sata to pci-e 6/8 pin converters for the risers? Perhaps there is a voltage drop which causes errors.

**Comment by Andriuzz66** [comment]
I use https://www.google.lt/search?q=risers+v007&dcr=0&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi8gLKn9OXXAhWIFZoKHXrHBQEQ_AUICigB&biw=1440&bih=697#imgrc=eLVAxxlZRUhlYM: 

these risers V007

**Comment by paucoma** [comment]
Hmm, these risers have data connections which are not recommended because in theory Sata 12v pin connections are rated for 1.5 Amps per circuit. There are 3 pin connections for the 12 volt rail which is a theoretical rating of 4.5 amps at 12 volts. Which is 54 Watts. Cards theoretically expect to be able to pull upto 75watts from the pci-e slot which could be the source of your problem. When drawing more than 54w there could be a voltage drop that will also vary with temperature a which the connector finds itself at. 
In summary I would recommend you use risers which have a pci-e 6 pin connector and use adapters. 2x Sata to pci-e 6 pin , or the 2x molex to pci-e 6 pin. I believe they are the version 006 or 006c version risers. 
Have a read here: https://www.reddit.com/r/EtherMining/wiki/hardware/risers

**Comment by Andriuzz66** [comment]
Thanks I will try now to power of several cards and will try to connect 1 riser to powersource instead 2 cards to one 6pin to power source. 

Will come back to you late this day.

**Comment by derubm** [comment]
just as a side comment: not changing power target might just be the problem;
at least try to get power target down to 64-70% somewhere. cards will run cooler and no (or minimal change in hashing speed will happen).  Also the Power draw will significantly drop. ( might solve your problem?) too bad you cannot change memory speed on your cards, as this is waste of hashes. nvidia cards get a really big advantage when mining eth on OC´d gfx ram.

**Comment by Andriuzz66** [comment]
First of all thx for idea. I will try it. The thing is I haven't boosted my cards. They run on auto. But I do not understand why 1) etherminer software stops every 45min-1hour aprox. and the pc (mining rig) also stops every several hours? I have connected 12 cards with 3x1kW power units, I have put PCI Risers v007. 32Mb RAM. fantastic PC but @#$%! jamming :) and have no more ideas. Now unplugged several vga cards and left only 9 will see if it jams... 
I need more suggestions and more Your experience.

**Comment by derubm** [comment]
first of all: are you running all your cards within 1 ethminer session?
if so,
1.  try to start 3 sessions a 4 cards. (maybe bundle them through ethproxy if needed)
2 . set verbosity to 1 ( reduces output lag in console ) (command -v 1 )
you will not any longer see the hashrate spam ,only submitted shares new blocks etc ..
3. ethproxy
if you for whatever reason want to see hashing speed in this setting, connect via ethproxy, it will show you hashing speed.
4. if all that does not help, - i know it´s unpopular for whatever reason, but install a rig with win 10 and try your luck there - i can tell you it might be better to mine on windows then on linux. ( you don´t even need to pay for windows, as you can always get a free copy from microsoft where you cant change background , but apart from that at least for mining 100 % functional. and you have the advantage of better supported drivers.

**Comment by Andriuzz66** [comment]
Thx for suggestion. Sorry if I am not profesional, but can you elaborate more with your steps :  
1) if I have 1 motherboard with 19 PCI's connections how can I start 3 sesions instead of 1?
2)What does that mean - set verbosity to 1 ( reduces output lag in console ) (command -v 1 )
Thanks in advance !!

**Comment by Andriuzz66** [comment]
@#$%^$@ made so meny changes and found out other issue, that this motherboard accepts only 13 vga cards. When I connect +1 (14) is crashes. So now started with 13cards to see if this issue of crashing will be over or not. Have no more ideas.

**Comment by paucoma** [comment]
To open a session with only a couple cards you can start a session with the command specifying the devices you want to use with "`--cuda-devices <number of devices seperated by spaces>`"

`>ethminer --cuda --cuda-devices 0 1 2 3 --farm http://<mining adrress>`

you can list all the devices that ethminer sees with 

```
>ethminer --list-devices --cuda
Listing CUDA devices.
FORMAT: [deviceID] deviceName
[0] GeForce GTX 1070
        Compute version: 6.1
        cudaDeviceProp::totalGlobalMem: 8589934592
...
```

Open several command windows and start one with 0,1,2,3 another with 4,5,6,7 etc...

**Comment by derubm** [comment]
for the verbosity thing: just add -v 1 to your startline.
(as mentioned above you will only see submited shares in the miner window itself, so dont expect it doing very much unless i submits a share or has to display new blocks etc.. , recommened only when you  are sure your cards are working well-or in case of crashes. be patient when you run ethminer this way, it might take a couple of seconds to see the first submitted share.)

**Comment by Andriuzz66** [comment]
Hi to all

to derubm: in which palce I have to add -v 1 ? with which program is better to mine ether: etherminer or Claymomor or else?

I made it, connected 13 cards and they are running to 2nd day. No find out that I will not be able to connect all 19 cards. It seem I have to mix and add more cards from Asus Radeon or something. Can someone suggest ?

BR

**Comment by derubm** [comment]
@Andriuzz66 : you can add -v 1 where ever you like in the start line for ethminer.
sample:
`start /B "ethminer" ethminer.exe -U -S stratum+tcp://hostaddress:port -O user:pass --farm-recheck 768 -SC 2 --cuda-parallel-hash 4 --cuda-grid-size 1280 --cuda-block-size 256 -v 2`

**Comment by Andriuzz66** [comment]
Thank for tips @derubm 
Can you suggest ethpool to mine in? Cause now mining in Ethereumpool.co and for 4 days do not receive payout.
BR

**Comment by derubm** [comment]
i ´d recommend miningpoolhub ;)

**Comment by DeadManWalkingTO** [comment]
1. Try Update Drivers.
Download and install the latests.

2. Try Update Ethminer.
Download the latest.

3. Try use -U for CUDA devices.
Test launch command:
`ethminer.exe -RH -U -S eu1.ethermine.org:4444 -FS us1.ethermine.org:4444 -O 0x7013275311fc37ccc1e40193D75086293eCb43A4.test`

4. Try to change P2 State and Power managment mode.
You can use [NVidiaProfileInspectorDmW](https://github.com/DeadManWalkingTO/NVidiaProfileInspectorDmW).
For the best mining hashrate choose from sector "5 - Common":
    - CUDA - Force P2 State (Set to "Off")
    - Power managment mode (Set to "Prefer maximum performance")

5. Try Tweak Win10.
You can use [Windows10MiningTweaksDmW](https://github.com/DeadManWalkingTO/Windows10MiningTweaksDmW) (#695).

6. Try Optimize/Overclock GPUs. 
You can use [MSI Afterburner](https://www.msi.com/page/afterburner) for GPU OverClock/Optimize. 

7. Try use a WatchDog
You can use [ETHminerWatchDogDmW](https://github.com/DeadManWalkingTO/ETHminerWatchDogDmW) (#735).

Please feedback.
Thank you!
